src/synthesized_insight/metrics/statistical_tests.py

```diff
@@ -114,8 +114,7 @@ def __init__(self,
     @classmethod
     def check_column_types(cls, check: ColumnCheck, sr_a: pd.Series, sr_b: pd.Series) -> bool:
         # Given columns should be both categorical or both ordinal
-        if (check.ordinal(sr_a) and check.ordinal(sr_b))\
-           or (check.categorical(sr_a) and check.categorical(sr_b)):
+        if (check.ordinal(sr_a) and check.ordinal(sr_b)):
```

Comment:
Not sure why categorical columns are allowed here, it's an ordinal measure

Reply:
In the SDK version, categoricals are allowed, but I think it is fair only to allow it for ordinal